.github/actions/integration-tests/action.yml

```diff
@@ -0,0 +1,121 @@
+name: Run integration tests
+description: Runs an integration test suite against an existing preview environment
+
+inputs:
+  test_suite:
+    description: "Test suite to run"
+    default: workspace
+  notify_slack_webhook:
+    description: "Optional Slack webhook to notify on test success/failure"
+    default: ""
+  preview_name:
+    description: "Name of the preview environment to run the tests against"
+    required: true
+  sa_key:
+    description: "The service account key to use when authenticating with GCP"
+    required: true
+  github_token:
+    description: "The GitHub token to use when authenticating with GitHub"
+    required: true
+  latest_ide_version:
+    description: "Use the latest IDE version"
+    required: false
+    default: ""
+  test_build_id:
+    description: "The build ID of the test run. Used in the IDE integration tests."
```

Comment:
Is it only IDE integration tests?

Reply:
yeah only here: https://github.com/gitpod-io/gitpod/blob/cd24ef2a15796e139fff0ab9b26aafcb786ff521/test/tests/ide/jetbrains/gateway_test.go#L169